validator/client/src/main/java/tech/pegasys/teku/validator/client/signer/SigningRequestBody.java

```diff
@@ -13,28 +13,51 @@
 
 package tech.pegasys.teku.validator.client.signer;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonGetter;
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.LinkedHashMap;
+import java.util.Map;
 import org.apache.tuweni.bytes.Bytes;
 
-@JsonIgnoreProperties(ignoreUnknown = true)
 public class SigningRequestBody {
+  @JsonProperty("signingRoot")
+  private Bytes signingRoot;
 
-  private final Bytes data;
+  @JsonProperty("type")
+  private SignType type;
 
-  @JsonCreator
-  public SigningRequestBody(@JsonProperty("data") final String data) {
-    this.data = Bytes.fromHexString(data);
+  @JsonAnySetter private final Map<String, Object> metadata = new LinkedHashMap<>();
```

Comment:
Any particular reason to used a `LinkedHashMap` instead of just `HashMap` here?

Reply:
Not for any particular performance reason I recall, I can switch it back to HashMap.

Reply:
Simpler is better. :). The only reason I can think of is if you're asserting the order of the content somehow in tests.  As long as you're using `.equals` comparison of two `HashMap` instances they won't care about order.

Reply:
The tests are asserting only outgoing json data in integration test, so using HashMap is working there.